---
title: Der Catch-Block wird niemals erreicht; "<exception>" wird oben in der gleichen Try-Anweisung behandelt
ms.date: 07/20/2015
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
ms.openlocfilehash: 987a509b4b099a0a1f728f55dfb2b35a8770af3f
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 06/04/2020
ms.locfileid: "84395479"
---
# <a name="catch-block-never-reached-exception-handled-above-in-the-same-try-statement"></a><span data-ttu-id="6981c-102">Der Catch-Block wird niemals erreicht; "\<exception>" wird oben in der gleichen Try-Anweisung behandelt</span><span class="sxs-lookup"><span data-stu-id="6981c-102">'Catch' block never reached; \<exception> handled above in the same 'Try' statement</span></span>
<span data-ttu-id="6981c-103">Ein `Catch` -Block im Code kann nicht erreicht werden, weil er in einem vorhergehenden `Try` -Block behandelt wird.</span><span class="sxs-lookup"><span data-stu-id="6981c-103">A `Catch` block in the code cannot be reached because it is handled in a preceding `Try` block.</span></span>  
  
<span data-ttu-id="6981c-104">Standardmäßig ist diese Meldung eine Warnung.</span><span class="sxs-lookup"><span data-stu-id="6981c-104">By default, this message is a warning.</span></span> <span data-ttu-id="6981c-105">Weitere Informationen zum Ausblenden von Warnungen oder zum Behandeln von Warnungen als Fehler finden Sie unter [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span><span class="sxs-lookup"><span data-stu-id="6981c-105">For more information about hiding warnings or treating warnings as errors, see [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).</span></span>
  
 <span data-ttu-id="6981c-106">**Fehler-ID:** BC42031</span><span class="sxs-lookup"><span data-stu-id="6981c-106">**Error ID:** BC42031</span></span>  
  
## <a name="to-correct-this-error"></a><span data-ttu-id="6981c-107">So beheben Sie diesen Fehler</span><span class="sxs-lookup"><span data-stu-id="6981c-107">To correct this error</span></span>  
  
- <span data-ttu-id="6981c-108">Entfernen Sie die redundante Anweisung.</span><span class="sxs-lookup"><span data-stu-id="6981c-108">Remove the redundant statement.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="6981c-109">Weitere Informationen</span><span class="sxs-lookup"><span data-stu-id="6981c-109">See also</span></span>

- [<span data-ttu-id="6981c-110">Verwenden des try/catch-Blocks zum Abfangen von Ausnahmen</span><span class="sxs-lookup"><span data-stu-id="6981c-110">How to use the try/catch block to catch exceptions</span></span>](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [<span data-ttu-id="6981c-111">Verwenden benutzergefilterter Ausnahmehandler</span><span class="sxs-lookup"><span data-stu-id="6981c-111">Using User-Filtered Exception Handlers</span></span>](../../standard/exceptions/using-user-filtered-exception-handlers.md)
- [<span data-ttu-id="6981c-112">Try...Catch...Finally-Anweisung</span><span class="sxs-lookup"><span data-stu-id="6981c-112">Try...Catch...Finally Statement</span></span>](../language-reference/statements/try-catch-finally-statement.md)
